blazor/gantt-chart/resource-view.md

```diff
@@ -167,6 +168,151 @@ To highlight the range of overallocation dates with a square bracket, you can en
 
 <!-- {% previewsample "https://blazorplayground.syncfusion.com/embed/rXVqNQsPpofLwCPA?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %} -->
 
+## Enable taskbar drag and drop

```

Comment:
Enable taskbar drag and drop between resources

Reply:
Updated